.github/mergify.yml

```diff
@@ -13,6 +13,7 @@ pull_request_rules:
       - base=master
       - -title~=(?i)\[wip\]
       - "label=PR | automerge"
+      - "label!=PR | message from pull request"
```

Comment:
What's the meaning of this rule? @scsven

Reply:
"PR | message from pull request" is ONLY used for the rule "Use the title and body from the pull request itself as the commit message", not "automatic merge for master when ci passed and the title does not contain “[WIP]".

This line of code is used to ensure that the two rules will not overlap.